### PR TITLE
Label system refactor

### PR DESCRIPTION
**Pull request by Fahasor**
## About the PR

This PR is a prolongation of #19082
Now labels text don't shown in name of entity (but they still shown in description, it already coded by someone)
Labels don't shown in name field, but they shown in vendomates.
Also some small fixes like adding suffix to jugs and making label text lowercase.

**Media**

https://github.com/space-wizards/space-station-14/assets/70820551/73ab51dc-ecec-4dea-926b-347472b08fbd

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

Not important i think

### CODE REVIEW

**Comment by juliangiebel** [comment]
Please don't close and open a new PR because you forgot to commit something. You can use your original one https://github.com/space-wizards/space-station-14/pull/19276

Also change the CL

**Comment by Fahasor** [comment]
So, i tried resolve merge conflict, but can't reproduce it on my local branch. Merge just goes by a normal way. Branches is up to date with SS14 original repository.

**Comment by Fahasor** [comment]
Hah, this is not so hard, it just auto merged, but i was confused because git does not show merge conflict to me, but github shows it (shit, this now does not looks like good PR, because it have already merged commit)

**Comment by Fahasor** [comment]
Is there something bad with CL, then give me feedback please, i think it's normal now

File: Content.Server/Labels/Label/LabelSystem.cs [hunk 1]
```diff
@@ -50,21 +51,14 @@ public void Label(EntityUid uid, string? text, MetaDataComponent? metadata = nul
 
             if (string.IsNullOrEmpty(text))
             {
-                if (label.OriginalName is null)
-                    return;
-
                 // Remove label
-                _metaData.SetEntityName(uid, label.OriginalName, metadata);
```
Comment (EmoGarbage404) [inline Content.Server/Labels/Label/LabelSystem.cs:57]: this removes the functionality of the labels changing the name of the entity, which i feel should be preserved

File: Content.Server/Labels/Label/LabelSystem.cs
Comment (AJCM-git) [inline Content.Server/Labels/Label/LabelSystem.cs:1]: yeah dont remove changing the entity name, if you *really* want to get rid of OriginalName, what you could do is scanning for the CurrentLabel in the name string and removing only that when deleting a label, but that may be very scuffed/unreadable, idk

**Review by AJCM-git — CHANGES_REQUESTED** [review]
Sorry i took so long, but now i have time to deal with this one. Everything but the one review i have is perfect

**Comment by metalgearsloth** [comment]
Re-open if you come back to this, it looks like something went funky with your merge.

**Pull request closed without merging**